### PR TITLE
update lang file to latest version

### PR DESCRIPTION
**Pull request by black23**
⚠️⚠️⚠️ Since we do not accept all types of pull requests and do not want to waste your time. Please be sure that you have read pull request rules:
https://github.com/louislam/uptime-kuma/blob/master/CONTRIBUTING.md#can-i-create-a-pull-request-for-uptime-kuma

Tick the checkbox if you understand [x]: 
- [x ] I have read and understand the pull request rules.

# Description

Fixes #(issue)

## Type of change

Please delete any options that are not relevant.

- Translation update

## Checklist

- [x ] My code follows the style guidelines of this project
- [ ] I ran ESLint and other linters for modified files
- [ x] I have performed a self-review of my own code and tested it
- [ ] I have commented my code, particularly in hard-to-understand areas
  (including JSDoc for methods)
- [ ] My changes generate no new warnings
- [ ] My code needed automated testing. I have added them (this is optional task)

## Screenshots (if any)

Please do not use any external image service. Instead, just paste in or drag and drop the image here, and it will be uploaded automatically.

### CODE REVIEW

**Comment by louislam** [comment]
`
/home/runner/work/uptime-kuma/uptime-kuma/src/languages/cs-CZ.js
  246:4  error  Parsing error: Unexpected token, expected "," (246:4)
`

File: src/languages/cs-CZ.js [hunk 1]
```diff
@@ -224,6 +242,7 @@ export default {
     teams: "Microsoft Teams",
     "Webhook URL": "URL adresa webhooku",
     wayToGetTeamsURL: "Informace o tom, jak vytvořit URL adresu webhooku naleznete {0}.",
+    wayToGetZohoCliqURL: "Informace o tom, jak vytvořit URL adresu webhooku naleznete {0}."
```
Comment (YehowahLiu) [inline src/languages/cs-CZ.js:245]:
```suggestion
    wayToGetZohoCliqURL: "Informace o tom, jak vytvořit URL adresu webhooku naleznete {0}.",
```

**Review by YehowahLiu — CHANGES_REQUESTED** [review]

**Comment by louislam** [comment]
#2659 

Oh no, I just realized that you guys translated a same language file at the same time. 

@Buchtic @black23

**Comment by louislam** [comment]
Since there are a lot of conflicts and we are switching to Weblate, I would suggest that it should be handled on Weblate later.

**Comment by black23** [comment]
I just write comment about Weblate :-) It's logcal step, best for collaboration.

**Comment by louislam** [comment]
> I just write comment about Weblate :-) It's logical step, best for collaboration.

Just finished setup Weblate, you could try to copy your translated word on https://weblate.kuma.pet. Sorry for the inconvenience.

**Comment by black23** [comment]
Great news! Lets go translate!

Thank you

út 24. 1. 2023 v 10:03 odesílatel Louis Lam ***@***.***>
napsal:

**Comment by louislam** [comment]
I saw you had worked on Weblate, thank you so much!

**Pull request closed without merging**